### PR TITLE
898/radio button circle clicks fail

### PR DESCRIPTION
**Pull request by eriklharper**
**Related Issue:** #898

### CODE REVIEW

**Comment by eriklharper** [comment]
> Is the embed prop still necessary if we are moving this to the default layout anyway? I guess I'm still confused how embed is different than inline 😁

I took out the embed prop in favor of the new `disable-spacing` prop.

**Comment by eriklharper** [comment]
> This is kind of a shadow dom oddity - do we want to keep it on the element a consumer placed it on and not pass it to the rendered child element, or do it this way? This way means folks may be looking for an id on the Calcite-input element they placed it on and not find it - then have to traverse to the wrapped input to find it?

I took this out since it turns out the default behavior doesn't affect the bug itself.

File: src/components/calcite-radio-button/calcite-radio-button.tsx [hunk 1]
```diff
@@ -187,55 +188,61 @@ export class CalciteRadioButton {
   //--------------------------------------------------------------------------
 
   @Listen("click")
-  check() {
+  check(event: MouseEvent | FocusEvent): void {
+    // Prevent parent label from clicking the first radio when calcite-radio-button is clicked
+    if (this.el.closest("label") && event.target === this.el) {
+      event.preventDefault();
+    }
```
Comment (eriklharper) [inline src/components/calcite-radio-button/calcite-radio-button.tsx:195]: This change was the kicker.

**Review by macandcheese — APPROVED** [review]
LGTM - let's get this in there!

**Pull request merged**